packages/react-dropdown/src/index.js

```diff
@@ -156,22 +158,26 @@ const Dropdown = ({
                       })
                     }
                   </Reference>
-                  {isOpen && (
-                    <DropdownList
-                      ref={ref}
-                      style={style}
-                      twoColumn={twoColumn}
-                      items={items}
-                      categories={categories}
-                      inputValue={inputValue}
-                      getItemProps={getItemProps}
-                      useFilter={useFilter}
-                      filterFn={filterFn}
-                      highlightedIndex={highlightedIndex}
-                      selectedItems={selectedItems}
-                      highlight={highlight}
-                    />
-                  )}
+                  {isOpen &&
+                    renderDropdown({
+                      dropdown: (
```

Comment:
I would have preferred to provide a component rather than a node, so that people could add their own properties to it, but this generates a render loop... if we ever decide we want to support this we can maintain backward compatibility and add a new `Dropdown` property.

Reply:
Actually, I'll try to spend few more minutes to figure out a solution right now.